spec/braze_ruby/rest/scheduled_broadcasts_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe BrazeRuby::REST::ScheduledBroadcasts do
+  let(:http) { double(:http) }
+  let(:end_time) { Time.now.iso8601 }
+
+  let(:payload) {
+    {
+      end_time: :end_time
+    }
+  }
+
+  let(:api_key) { :api_key }
+
+  subject { described_class.new(api_key, :rest_url, {}, **payload) }
+
+  before { subject.http = http }
+
+  it "makes an http call to the scheduled broadcasts endpoint" do
+    expect_scheduled_broadcasts_http_call
+
+    subject.perform
+  end
+
+  def expect_scheduled_broadcasts_http_call
```

Comment:
I'm curious to know more about why a method is extracted here - why not have this expectation inside the body of the it block?

Reply:
I was honestly just copying another spec that had similar breadth.

There are quite a few opportunities to standardize patterns in this library. That might be something I look into next!

Reply:
Ohhh, I just opened the first file I found and it happened to NOT have a method extracted so yeah I see what you mean about standardizing. No problem!